screenshots/test.py

```diff
@@ -0,0 +1,57 @@
+from seltest import url, waitfor, hide, Base
+from selenium.webdriver.support.ui import Select, WebDriverWait
+import sys
+
+PILEUP_RENDERED_JS = '''
+return document.querySelector('.track.pileup canvas') && document.querySelector('.track.pileup canvas').height > 100
+'''
+
+def _pileup_is_rendered(driver):
+    return driver.execute_script(PILEUP_RENDERED_JS)
+
+
+def wait_for_pileup(driver):
+    driver.implicitly_wait(0)
```

Comment:
Hmm, for my own edification, why do you need to wait for 0 seconds? 

Definitely need to add this to seltest as a decorator.